core/templates/pages/exploration-editor-page/translation-tab/services/translation-language.service.ts

```diff
@@ -28,7 +28,7 @@ import { LoggerService } from 'services/contextual/logger.service';
   providedIn: 'root'
 })
 export class TranslationLanguageService {
-  private activeLanguageCode: string = null;
+  private activeLanguageCode!: string;
```

Comment:
Could you add a comment here on why `!` is needed?

Reply:
done